### PR TITLE
Organize README with updated information

### PR DESCRIPTION
**Pull request by jack11wagner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,74 @@
...
+### Disclaimers
+--------
+"Regulations.gov and the Federal government cannot verify and are not responsible for the accuracy or authenticity of the data or analyses derived from the data after the data has been retrieved from Regulations.gov."
+
+In other words, "once the data has been downloaded from Regulations.gov, the U.S. Government cannot verify and is not responsible for the quality, accuracy, reliability, or timeliness of any analyses conducted using the downloaded data."
+
+This product uses the Regulations.gov Data API but is neither endorsed nor certified by Regulations.gov.
+
+--------
+This project is currently being developed by a student research team at Moravian College in association with Careset under Fred Trotter.
```
Comment (jack11wagner) [inline README.md:53]: Remove

**Comment by justinszaro** [comment]
Other than a quick switch from college to university, this looks fantastic! Are we no longer displaying the architecture.png?

**Comment by bryantqr** [comment]
I agree with Justin, this looks fantastic. You added a lot of important information that wasn't there. Great job!

**Comment by pereze03** [comment]
The information added is especially important for anyone outside the project trying to contribute.
I read through everything and it looks good to be merged. Well done.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,75 @@
-# Capstone2021
+# Mirrulations
 
-A client/server application to download [`Regulations.gov`](https://www.regulations.gov/).
+Software to create and maintain a mirror of regulations.gov
```
[the remark this answers is not in the document]
Comment (bryantqr) [reply README.md:3]: I think the reason @jack11wagner is not including that is because he added that to the regulations.gov part

**Pull request merged**